gap/oper.gi

```diff
@@ -166,7 +166,13 @@ end);
 
 InstallMethod(DigraphRemoveVertices, "for an immutable digraph and a list",
 [IsImmutableDigraph, IsList],
-{D, list} -> MakeImmutable(DigraphRemoveVertices(DigraphMutableCopy(D), list)));
+function(D, list)
+  if ForAll(list, IsPosInt) and
+      not (ForAny(list, x -> x <= DigraphNrVertices(D))) then
```

Comment:
If the list has negative integers, then presumably the input digraph should still not be copied?

Instead of testing `IsPosInt` and then checking `<=`, would we be better to check `ForAny(list, x -> not x in DigraphVertices(D)`?  Then it's a single clean statement that catches everything.